docs/md-vs-ipynb.md

```diff
@@ -0,0 +1,40 @@
+# How to choose between `.ipynb` and `.md` when writing Jupyter notebooks
```

Comment:
Again, a note to self for now:

I somewhat want us to make a simple rule of thumb:

- In version control? MD file
- Otherwise? ipynb

This will become much easier to push once our ipynb round-tripping is more robust